marklogic-data-hub-central/ui/e2e/cypress/e2e/modeling/writerScenarioGraph.cy.tsx

```diff
@@ -478,4 +478,38 @@ describe("Entity Modeling: Graph View", () => {
     });
     graphViewSidePanel.getPropertyName("purchased").should("not.exist");
   });
+
+  it("edit structured property relationship", () => {
+    toolbar.getModelToolbarIcon().click();
+    modelPage.selectView("table");
+
+    //Creates a structured property on Customer
+    entityTypeTable.viewEntityInGraphView("Customer");
+    graphViewSidePanel.getAddPropertyForStructuredProperties("shipping").click();
+    propertyModal.newPropertyName("structuredPropertyTest");
+    propertyModal.openPropertyDropdown();
+    propertyModal.getTypeFromDropdown("Related Entity");
+    propertyModal.getCascadedTypeFromDropdown("Person");
+    propertyModal.getSubmitButton().click();
+    cy.waitForAsyncRequest();
+
+    //Edits the relationship from the graph
+    graphVis.getPositionOfEdgeBetween("Customer,Person").then((edgePosition: any) => {
+      cy.wait(150);
+      graphVis.getGraphVisCanvas().click(edgePosition.x, edgePosition.y, {force: true});
+    });
+    relationshipModal.editRelationshipName("newRelationshipName");
```

Comment:
@littlecherro Can we also add validations where we edit names with special characters like using hyphen etc in the relationship name and check if everything works fine as we had some issues with special characters in multiple places earlier.

Reply:
@ngodugu-marklogic done :)